tests/integration/testdata/sync/code/before/template-python-no-dependencies.yaml

```diff
@@ -11,4 +11,4 @@ Resources:
     Properties:
       CodeUri: python_function_no_deps/
       Handler: app.lambda_handler
-      Runtime: python3.9
```

Comment:
Why does 3.9 not work?

Reply:
Windows canaries doesn't have 3.9 configured with it, so we need to use 3.7 until we update configuration there.